### DISCUSSION TITLE
Agent bots at self-hosted instance

### DISCUSSION THREAD

**Discussion by bernapastorini** [chatwoot/chatwoot#6303] [Self hosted]
Hi! We have a self-hosted Chatwoot instance in Kubernetes using version **2.13.0** and we want to use the agent bots. 

In the settings menu does not appear the option "Bots" as it does when accessing Chatwoot as SaaS, and when accessing directly by URL to create a new one, does not appear the text field to write CSML.

Does the **CSML_EDITOR_HOST** variable have something to do with this? What value should it have?

I appreciate any help or guidance.

Greetings!

**Comment by wuorena** [comment]
I'm facing the same problem, did you had any luck with this?

**Reply by bernapastorini** [reply]
Unfortunately, not yet 😥

**Comment by wrapper** [comment]
Are there any updates on this? Also looking to use this functionality. cc @pranavrajs
